Zend/zend_API.h

```diff
@@ -104,6 +104,8 @@ typedef struct _zend_fcall_info_cache {
 #define ZEND_ARG_CALLABLE_INFO(pass_by_ref, name, allow_null)        { #name, ZEND_TYPE_ENCODE(IS_CALLABLE, allow_null), pass_by_ref, 0 },
 #define ZEND_ARG_TYPE_INFO(pass_by_ref, name, type_hint, allow_null) { #name, ZEND_TYPE_ENCODE(type_hint, allow_null), pass_by_ref, 0 },
 #define ZEND_ARG_VARIADIC_INFO(pass_by_ref, name)                    { #name, 0, pass_by_ref, 1 },
+#define ZEND_ARG_VARIADIC_TYPE_INFO(pass_by_ref, name, type_hint)    { #name, ZEND_TYPE_ENCODE(type_hint, 0), pass_by_ref, 1 },
+#define ZEND_ARG_VARIADIC_OBJ_INFO(pass_by_ref, name, classname)     { #name, ZEND_TYPE_ENCODE_CLASS_CONST(#classname, 0), pass_by_ref, 1 },
```

Comment:
As the non-variadic variants include an allow_null parameter, I think it should be added here as well, for consistency.